content/tutorials/create-users-and-teams.md

```diff
@@ -12,40 +12,44 @@ draft: true
 
 {{% tutorials/step duration="1" title="Introduction" %}}
 
-Create users and organize them into teams. Manage resource access for each user and team through roles and folders.
+This tutorial is useful for admins and others who want to learn how to manage users. In this tutorial, you'll add multiple users, organize them into teams, and make sure they're only able to access the resources they need.
 
-### What you'll learn
 
-- How to add users
-- How to assign users to teams
-- How to manage dashboard access using folders
-- How to override access for individual dashboards
+### You'll learn to
 
-### What you'll build
+- Add users.
+- Assign users to teams.
+- Manage dashboard access using folders.
+- Override access for individual dashboards.
+
+### Scenario
 
 _Graphona_, a fictional telemarketing company, has asked you to configure Grafana for their teams.
 
-In this tutorial, you'll add multiple users, organize them into teams, and make sure they're only able to access the resources they need.
+### You will
+* Create users and organize them into teams. 
```

Comment:
I'm in favor of using hyphens, `-`, for unordered lists. It's the more popular variant, and doesn't require a modifier key on most keyboard.